Libplanet/Blockchain/Renderers/DelayedActionRenderer.cs

```diff
@@ -68,14 +69,22 @@ public class DelayedActionRenderer<T> : DelayedRenderer<T>, IActionRenderer<T>
         /// <param name="store">The same store to what <see cref="BlockChain{T}"/> uses.</param>
         /// <param name="confirmations">The required number of confirmations to recognize a block.
         /// See also the <see cref="DelayedRenderer{T}.Confirmations"/> property.</param>
-        public DelayedActionRenderer(IActionRenderer<T> renderer, IStore store, int confirmations)
+        /// <param name="reorgResistantHeight">Configures the height of blocks to maintain the
+        /// <see cref="ActionEvaluation"/> buffer. Buffered <see cref="ActionEvaluation"/>s
+        /// that belong to blocks older than this height from the tip are gone.</param>
```

Comment:
We seem to miss the explanation the meaning of the default value (0).

```suggestion
        /// that belong to blocks older than this height from the tip are gone.
        /// If zero, which is a default value, is passed the buffer is not cleared.</param>
```

Reply:
1c969e6 added